Updating the snowflake OAuth page (#6916)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/set-up-snowflake-oauth.md

To get started, copy the connection's redirect URI from dbt Cloud:
1. Navigate to **Account settings**.
1. Select **Projects** and choose a project from the list. 

> 1. Select the development connection to view its details and set the **OAuth method** to "Snowflake SSO".

Reviewer: ```suggestion
1. Click the **Development connection** field to view its details and set the **OAuth method** to "Snowflake SSO".
```

  integration_secrets;
```

> To complete the creation of your connection:

Reviewer: ```suggestion
To complete the creation of your connection in dbt Cloud:
```

> 1. From dbt Cloud's Account Settings, click **Connections** and select a connection.

Reviewer: ```suggestion
1. Navigate to your **Account Settings**, click **Connections**, and select a connection.
```
